app/views/posts/show.html.slim

```diff
@@ -17,9 +16,8 @@
       = Post.human_attribute_name(:description)
     = @post.description
   p
-    strong
-      = Post.human_attribute_name(:visibility)
-    = @post.visibility
+    strong Visibility
+    = Visibility.find(@post.visibility_id).name
```

Comment:
you should never do `find` in view.  Define an association  `belongs_to :visibility` in `post.rb`
When you define it, you will be able to use `@post.visibility.name`